webapp/src/main/resources/public/js/components/workbench/GitBlameInfoModal.js

```diff
@@ -43,6 +44,28 @@ let GitBlameInfoModal = React.createClass({
         );
     },
 
+    /**
+     * @param label                 The label for the data to display
+     * @param data                  The link text
+     * @param branchScreenshots     The screenshots for this branch
+     * @returns {*}                 The row of label:link to display in the modal
+     */
+    displayScreenshotLink(label, data, branchScreenshots) {
+        if (data == null || !branchScreenshots || !branchScreenshots.length) {
```

Comment:
looks like the label is always `"Screenshot"` and data is `"View"` all the time. The `branchScreenshots` can also be fetch in the function.

We usually internationalize the UI strings (though this page might be missing some)

```
displayScreenshotLink() {
   const label = this.props.intl.formatMessage({ id:  "...screenshot"});
...
onClick={() => this.props.onViewScreenshotClick(branchScreenshots)}
                    className="col-sm-9 git-blame-screenshot clickable">
                   <FormattedMessage id="....view" />
}
```